components/support/versioning/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "versioning"
+version = "0.1.0"
+authors = ["Bastian Gruber <bgruber@mozilla.com>"]
```

Comment:
This should probably include the author credit of whomever added this module to nimbus first, or `project-nimbus@` at the very least (the credit for nimbus isnt right either but thats for another day)